graphrag_sdk/orchestrator/steps/parallel.py

```
@@ -34,13 +57,29 @@ def output(self) -> str:
 
 
 class ParallelProperties:
+    """
+    Represents properties for parallel execution of steps.
+
+    Attributes:
+        steps (list[PlanStep]): The list of steps to execute in parallel.
+    """
+    
     steps: list["PlanStep"]
```

Ruff

Undefined name `PlanStep`

(F821)